src/containers/pages/MDAPoint/ClientListView/helpers/serviceHooks.ts

```diff
@@ -39,7 +39,7 @@ export const postUploadedFile = async (
   params?: string
 ) => {
   const bearer = `Bearer ${getAccessToken(store.getState())}`;
-  await fetch(`${OPENSRP_API_BASE_URL}/upload/?event_name=Child%20Registration${params}`, {
+  await fetch(`${OPENSRP_API_BASE_URL}upload/?event_name=Child%20Registration${params}`, {
```

Comment:
1.  Lets not have magic strings here
2. `Child%20Registration`  should not be hard coded.  It should be a variable/param